config/s2i/jenkins/master-downstream/plugins.txt

```diff
@@ -90,7 +90,7 @@ pipeline-model-extensions:1.2.7
 pipeline-rest-api:2.9
 pipeline-stage-step:2.3
 pipeline-stage-tags-metadata:1.2.7
-pipeline-stage-view:2.9
+pipeline-stage-view:2.10
```

Comment:
This is not even the plugins.txt for the Jenkins master that the unit tests run on. Furthermore, the master the unit tests *are* running on has pipeline-stage-view 2.10. This is not the fix

Reply:
where are the unit tests being run?

Reply:
https://jenkins-continuous-infra.apps.ci.centos.org/view/Linchpin/job/ci-linchpin-unittests/

Reply:
I mean the s2i for it :), but nvm we are good I guess, thanks!

Reply:
Well, the unit tests are run on the upstream master, like I posted, so if you go to the s2i for the upstream master and go to jobs, you find https://github.com/CentOS-PaaS-SIG/linchpin/blob/develop/config/s2i/jenkins/master-upstream/configuration/jobs/ci-linchpin-unittests/config.xml

Reply:
ahh, got it, thnx!